bitrepository-client/src/main/java/org/bitrepository/modify/putfile/BlockingPutFileClient.java

```diff
@@ -63,20 +63,13 @@ public BlockingPutFileClient(PutFileClient client) {
      * @return The list of {@link ContributorEvent}s received during the operation
      * @throws OperationFailedException The operation didn't complete successfully.
      */
-    public List<ContributorEvent> putFile(
-            String collectionID,
-            URL url,
-            String fileID,
-            long sizeOfFile,
-            ChecksumDataForFileTYPE checksumForValidationAtPillar,
-            ChecksumSpecTYPE checksumRequestsForValidation,
-            EventHandler eventHandler,
-            String auditTrailInformation)
-            throws OperationFailedException {
+    public List<ContributorEvent> putFile(String collectionID, URL url, String fileID, long sizeOfFile,
+                                          ChecksumDataForFileTYPE checksumForValidationAtPillar,
+                                          ChecksumSpecTYPE checksumRequestsForValidation, EventHandler eventHandler,
+                                          String auditTrailInformation) throws OperationFailedException {
```

Comment:
Nit-picking, I know: my preference would be for still keeping the throws clause on a separate line.

Reply:
To me, the way this is done is what is easiest to read and looks the prettiest, but I think this is an opinionated matter, so  this would be a great opportunity for one of the things we should discuss along with the code style guidelines

Reply:
It's certainly opinionated. I'm happy to leave this as it is for now. And will be happy to take part in the code style discussion if it happens before I leave the project.

Reply:
I believe that the architectural council is possibly gonna issue an official code style guide soon, I think we should look into that as well